test/unittest/unittest_nntrainer_layers.cpp

```diff
@@ -495,6 +495,9 @@ TEST(nntrainer_Conv2DLayer, forwarding_01_p) {
   EXPECT_EQ(status, ML_ERROR_NONE);
   layer.setInputDimension(previous_dim);
 
+  nntrainer::ActivationLayer actLayer;
+  actLayer.setActivation(nntrainer::ACT_SIGMOID);
+
```

Comment:
This is not directly related with this PR, but we may need unit test case for the tizen api about activation layer.

Reply:
I think we need a discussion in #210. Thanks for pointing it out :)